kernel/pom.xml

```diff
@@ -532,20 +483,6 @@
           <warSourceDirectory>${basedir}/src/main/webapp</warSourceDirectory>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
I'd like to see this one updated and attached to all projects so that source is able to be attached in eclipse. Could we look at that instead of removing?